src/lib/gundb/UserStorageClass.js

```diff
@@ -834,10 +836,12 @@ export class UserStorage {
 
       //mint event is probably bridge
       if (feedEvent.type === EVENT_TYPE_MINT && receipt.status) {
-        updatedFeedEvent.data.reason = 'Your Transfereed G$s'
+        updatedFeedEvent.data.reason = 'Your Transferred G$s'
         updatedFeedEvent.data.customName = 'Bridge'
       }
 
+      updatedFeedEvent = await this.setFeedEventProfileFields(updatedFeedEvent)
```

Comment:
instead of calling this form multiple places, can we just call it in the updateFeedEvent method?

Reply:
done